server/events/event_parser_test.go

```diff
@@ -1232,6 +1233,7 @@ func TestParseAzureDevopsRepo(t *testing.T) {
 		VCSHost: models.VCSHost{
 			Hostname: "dev.azure.com",
 			Type:     models.AzureDevops,
+			VcsToken: "azuredevops-token",
```

Comment:
Should this token be generated within Atlantis instead of hard coded?

If it has to be hard coded, couldn't it be assigned to a variable and reused?

Reply:
The token is generated within Atlantis - it's a base64 derivative of the vcsToken - which is the PAT token supplied to **azuredevops-token** at startup.